CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 Your contributions are always welcome!
 
+<h3>It&rsquo;s Hacktoberfest 2022 @Meilisearch</h3>
+<p><a href="https://hacktoberfest.com">Hacktoberfest</a> is a celebration of the open-source community. This year, and for the third time in a row, Meilisearch is participating in this fantastic event.</p>
+<p>You&rsquo;d like to contribute? Don&rsquo;t hesitate to check out our <a href="https://github.com/meilisearch/meilisearch-ruby/blob/main/CONTRIBUTING.md">contributing guidelines</a>.<br /><br />And the longer, more complete guideline documentation you can follow <a href="https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md">this link</a>.</p>
```

Comment:
it's a link to the Ruby integration repo

Reply:
I also would put this section in the README since it's the first think a person sees...

Reply:
I'll move it to the readMe and change the link!

Reply:
Could you update your PR accordingly? thanks